dist_app_linux.tcjs

```diff
@@ -1,21 +1,20 @@
 let tc = TCF.define(TCF.CPP_TRANSFORM);
-tc.pocoLoc = '/home/tests/poco';
+//tc.pocoLoc = '/home/tests/poco';
 tc.sources = [
     'platform:/resource/DistributedPingPong/CPPModel.emx#_qJc4wfMMEeiMwvLnGvgCsg'
 ];
 tc.createTargetProject = true;
 tc.targetProject = '/DistributedPingPong_target';
 tc.prerequisites = [
-    'platform:/resource/LibTcpServer/tcpServerLib.tcjs'
+    'platform:/resource/LibTcpServer/tcpServerLib_linux.tcjs'
 ];
 tc.compilationMakeType = MakeType.GNU_make;
-tc.compileArguments = '$(DEBUG_TAG) -fexceptions';
+tc.compileArguments = '$(DEBUG_TAG)';
 tc.inclusionPaths = [
-    tc.pocoLoc + '/Net/include',
-    tc.pocoLoc + '/JSON/include',
-    tc.pocoLoc + '/Foundation/include'
+
 ];
 tc.targetConfiguration = 'LinuxT.x64-gcc-12.x';
 tc.targetServicesLibrary = '${RSA_RT_HOME}/C++/TargetRTS';
 tc.topCapsule = 'platform:/resource/DistributedPingPong/CPPModel.emx#_vAzCMPMMEeiMwvLnGvgCsg';
-tc.linkArguments = '-L' + tc.pocoLoc + '/lib/Linux/x86_64/ -lPocoJSON -lPocoNet -lPocoFoundation';
\ No newline at end of file
+//tc.linkArguments = '-L' + tc.pocoLoc + '/lib/Linux/x86_64/ -lPocoJSON -lPocoNet -lPocoFoundation';
+tc.cppCodeStandard = 'C++ 98';
```

Comment:
In tcp server lib you are using nullptr and brace initialization. How does this match with this old code standard setting?
Why do we define codeStandard?

Reply:
ok, I will remove it. We have decided to follow C++11 standard.